### ISSUE TITLE
Current code allows datetime without timezone specification

### ISSUE THREAD

**Issue by Snawoot** [EFForg/starttls-policy-cli#13]
For example tests/policy_test.py contains following:
```python
test_json = '{\
    "author": "Electronic Frontier Foundation",\
        "expires": "2014-05-26T01:35:33",\
        "timestamp": "2014-05-26T01:35:33",\
        "policies": {\
            ".valid.example-recipient.com": {\
                "min-tls-version": "TLSv1.1",\
                    "mode": "enforce",\
                    "mxs": [".valid.example-recipient.com"],\
                    "tls-report": "https://tls-rpt.example-recipient.org/api/report"\
            }\
        }\
    }'
```

Such test json is used only for testing of Config container class and problem here doesn't breaks anything, but it is a good example of a problem.

Here `expires` and `timestamp` are specified without timezone. `dateutil.parse` of it will result timezone-unaware datetime object, which we can't compare to timezone-aware objects. Spec of list format _seems_ to require timezone in string form: `yyyy-MM-dd'T'HH:mm:ssZZZZ`.

What is best way to handle such records without TZ? I see two options:
* Reject configs with any datetime without TZ specification.
* Threat such datetime records as UTC.

I think second option is more liberal to list maintainers and end users. Please advise.

**Comment by sydneyli** [comment]
I agree that we should treat non-TZ datetime records as UTC, and specify this requirement in the documentation.

**Comment by Snawoot** [comment]
@sydneyli probably if we will [enforce policy structure with schema](https://github.com/EFForg/starttls-everywhere/pull/98), such cases will be not allowed to exist.

Anyway, I'll adjust date parsing routine to always return proper TZ-aware datetime and assume UTC if no TZ can be inferred from parsed date format.

**Comment by sydneyli** [comment]
Fixed in #20.
